libcrux-ml-dsa/src/simd/avx2/arithmetic.rs

```diff
@@ -56,6 +56,11 @@ pub(super) fn montgomery_multiply_by_constant(lhs: Vec256, constant: i32) -> Vec
     res
 }
 
+#[hax_lib::fstar::before(
```

Comment:
It would be wonderful to get to a readable annotation here.
I always disliked the tactic calls in F* and would hate to expose even more inscrutable syntax here.

Reply:
Sure, I can add a `postprocess_with` macro